### DISCUSSION TITLE
Native python ZigZag code

### DISCUSSION THREAD

**Discussion by vincehayek** [DaveSkender/Stock.Indicators#969] [Q&A] [answered]
Hello,
By any chance, do you have the native python code for the ZigZag indicator ?
I'm experiencing some issues when multithreading in Python, might come from the wrapper.

Otherwise I'll try to translate it from C#, I think the code source is here.
https://github.com/DaveSkender/Stock.Indicators/blob/main/src/s-z/ZigZag/ZigZag.Series.cs

Cheers

**Comment by DaveSkender** [comment, marked as answer]
Not yet.  Our Python project is just a wrapper that calls the NuGet DLL built in this core project.
We do have ambition to slowly replace with native code without changing the API but have not started that yet.

**Reply by vincehayek** [reply]
Ok, I'll work on it and share the result if I deliver something.
But I'm not a great programmer, won't be at your level!
